src/web/app/pages-instructor/student-profile/student-profile.ts

```diff
@@ -12,3 +13,12 @@ export interface StudentProfile {
   moreInfo: string;
   pictureKey: string;
 }
+
+/**
+ * Contains a student's details.
+ */
+export interface StudentDetails {
```

Comment:
Why do you need this?

Reply:
because the legacy api /accounts GET. The response DTO is not there yet due to its complexity. Will be removed after refactor this api in later tasks.